lib/tool_belt/commands/check_deprecation_warnings.rb

```diff
@@ -0,0 +1,68 @@
+require 'parser/current'
+require_relative '../deprecation_warning_parser.rb'
+
+module ToolBelt
+  module Command
+    class CheckDeprecationWarningsCommand < Clamp::Command
```

Comment:
I'd like for this command class to be dumber than it is now: basically just taking the version and path inputs and passing them to another class for processing. That's more or less what the other commands do in this repo.

ie `scan_for_deprecations` should be somewhere else and might get called like this:
```
<snip>

outdated_deprecations = paths_to_scan.flat_map |file|
  finder = ToolBelt::DeprecationFinder.new(file)
  finder.for_version(version).select { |dep| dep[:outdated] == true }
end

# do the reporting,
```

I think messaging result text can stay in the command  since it's expected to give those messages. Let me know what you think